Jenkinsfile.release

```diff
@@ -0,0 +1,206 @@
+// -*- Groovy -*-
+/*
+  Require credentials:
+    - Microsoft Azure Service Principal
+    - Azure Storage Account
+
+  Maven docker image cannot be run with the root user
+  https://batmat.net/2017/01/30/do-not-run-your-tests-in-continuous-integration-with-the-root-user/
+  Docker container related to jenkins-jnlp require:
+  * uid 1000,
+  * gid 1000
+  * /home/jenkins/.jenkins
+
+  Jenkins Plugins:
+    * Azure-Credentials
+    * SSH-agent
+
+  docker image olblak/maven is just a customized docker image based on maven where maven is run as
+  user jenkins with uid 1000 and gid 1000, there is also a volume /home/jenkins/.jenkins defined in the image
+  otherwise the directory is created with root permission and Jenkins cannot add files there
+*/
+
+pipeline {
+    agent {
+      kubernetes {
+        label 'jenkins-release'
+        defaultContainer 'jnlp'
+        yaml '''
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    some-label: jenkins-release
+spec:
+  containers:
+  - name: maven
+    image: olblak/maven:3-jdk-8
+    imagePullPolicy: Always
+    workdir: /home/jenkins
+    env:
+      - name: HOME
+        value: "/home/jenkins"
+    command:
+    - cat
+    tty: true
+  - name: azure-cli
+    image: microsoft/azure-cli:2.0.59
+    workdir: /home/jenkins
+    command:
+    - cat
+    tty: true
+'''
+      }
+    }
+
+    options {
+      buildDiscarder logRotator(
+        artifactDaysToKeepStr: '',
+        artifactNumToKeepStr: '',
+        daysToKeepStr: '',
+        numToKeepStr: '10'
+      )
+      disableConcurrentBuilds()
+    }
+    triggers {
+      // Trigger a new release once a week on Sunday
+      cron('H H(0-7) * * 0')
```

Comment:
I think for a job of this importance, we can set a specific time of day, not rely on `H`!

Reply:
As a first iteration,I just remove this cron as  we'll trigger the release manually